tests/test_config.py

```diff
@@ -22,6 +22,16 @@ def alt_cfg_module():
     return 'config.test_cfg_alt'
 def cfg_file():
     return os.path.join(dirname, "config", "test_cfg.py")
+def pyc_cfg_file():
+    PY33 = (sys.version_info >= (3, 3))
```

Comment:
Can we use the `py_compile.compile()` function instead, so that this is totally portable and we don't have to check in bytecode?